modules/util/portal-tools-rest-builder/src/test/java/com/liferay/portal/tools/rest/builder/RESTBuilderTest.java

```diff
@@ -82,6 +84,20 @@ public void testCreateRESTBuilder() throws Exception {
 		Assert.assertFalse(sampleImplDir.exists());
 	}
 
+	private void _assertBooleanParameterName(String filesPath)
+		throws Exception {
+
+		File queryJavaFile = new File(
+			filesPath.concat(
+				"/sample-impl/src/main/java/com/example/sample/internal" +
+					"/resource/v1_0_0/BaseTestResourceImpl.java"));
+
+		String text = new String(
+			Files.readAllBytes(queryJavaFile.toPath()), StandardCharsets.UTF_8);
+
+		Assert.assertFalse(text.contains("Boolean booleanValue"));
```

Comment:
AssertFalse? Or assertTrue? I did not check the test, I am just doing a quick review.
I ask if this should be `assertTrue` because you are adding the string `Value` at the end of the parameterName, so if it is a `boolean` it should be `booleanValue` but maybe I'm missing something 😅

Reply:
ouch!

Reply:
Fixed! Thanks @LuismiBarcos!

Reply:
I push forced.